docs/CHANGELOG.asciidoc

```diff
@@ -90,10 +91,356 @@ For detailed information about the n.n.n release, review the following sections.
 
 --
 
+[[release-notes-7.12.0]]
+== {kib} 7.12.0
+
+coming::[7.12.0]
+
+For detailed information about the {kib} 7.12.0 release, review the following information.
+
+[float]
+[[breaking-changes-7.12.0]]
+=== Breaking change
+
+Breaking changes can prevent your application from optimal operation and performance. Before you upgrade to 7.12.0, review the breaking change, then mitigate the
+impact to your application.
+
+// The following section is re-used in the Installation and Upgrade Guide
+
+// tag::notable-breaking-changes[]
+
+[discrete]
+[[maps-7.12.0-breaking-change]]
+.Removes the Geo threshold alert type
+[%collapsible]
+====
+*Details:* +
+Geo containment alerts and the Geo threshold alert type have similar functionality, so the Geo threshold alert type has been removed.
+
+*Impact* +
+Use Geo containment alerts.
+
+For more information, refer to {kibana-pull}89632[#89632].
```

Comment:
In the Deprecations section, the line for more information is in the Details section.  Should the placement be consistent here?